### DISCUSSION TITLE
[ feature request ] [ GeoMap ] layer visible range

### DISCUSSION THREAD

**Discussion by yoyo421** [grafana/grafana#47588] [Dashboards, Panels & Visualizations]
hide & show layers based on the zoom level

**Comment by sesoko838** [comment]
Interested in this for handling significant number of markers (approx 2500)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
